src/plugins/saved_objects/README.md

```diff
@@ -0,0 +1,119 @@
+# Saved object
+
+The saved object plugin provides all the core services and functionalities of saved objects. It is utilized by many core plugins such as [`visualizaton`](../visualizations/), [`dashboard`](../dashboard/) and [`wizard`](../wizard/), as well as external plugins. Saved object is the primary way to store app and plugin data in a standardized form in OpenSearch Dashboards. They allow plugin developers to manage creating, saving, editing and retrieving data for the application. They can also make reference to other saved objects and have useful features out of the box, such as migrations and strict typings. The saved objects are managed by the Saved Object Management UI.
```

Comment:
> The saved objects are managed by the Saved Object Management UI.

NIT: Technically there is a saved objects API that OpenSearch Dashboards serves up. But this statement reads to me this the only way to interact with saved objects. So maybe we can mention at least "The saved objects can be managed by the Saved Object Management UI"